{{cookiecutter.project_name}}/.github/workflows/release.yml

```diff
@@ -14,24 +15,32 @@ jobs:
   release:
     runs-on: ubuntu-latest
     concurrency: release
+    permissions:
+      id-token: write  # IMPORTANT: this permission is mandatory for trusted publishing using PyPI
+
 
     if: ${{ github.ref == 'refs/heads/main' && github.event.workflow_run.conclusion == 'success'}}
     steps:
-      # Checkout action is required for token to persist
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
           token: ${{ secrets.PAT }}
 
       - name: Python Semantic Release
-        uses: relekang/python-semantic-release@v7.33.1
+        id: release
+        uses: python-semantic-release/python-semantic-release@v8.0.4
+        with:
+          github_token: ${{ secrets.PAT }}
+
+      - name: Publish package distributions to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        if: steps.release.outputs.released == 'true'
+        # This action supports PyPI's trusted publishing implementation, which allows authentication to PyPI without a manually 
+        # configured API token or username/password combination. To perform trusted publishing with this action, your project's 
+        # publisher must already be configured on PyPI.
```

Comment:
Oh, my understanding is that users need to do less now, then?

Perhaps we can include a link on how to seutp PyPI to "trust a given GitHub repository and workflow"? https://blog.pypi.org/posts/2023-04-20-introducing-trusted-publishers/